### PR TITLE
added some missing tracks and fixed one that wasn't updated properly

### PR DESCRIPTION
**Pull request by owlisnotacat1**

### CODE REVIEW

File: data/Music/The Legend of Zelda Series/Breath of the Wild/Fanfare - Sit by the campfire.meta [hunk 1]
```diff
@@ -0,0 +1,3 @@
+TLoZ Breath of the Wild - Sit by the campfire
```
Comment (PurpleHato) [inline data/Music/The Legend of Zelda Series/Breath of the Wild/Fanfare - Sit by the campfire.meta:1]: The Legend of Zelda: Breath of the Wild - Sit by the campfire

**Review by PurpleHato — CHANGES_REQUESTED** [review]
Please, use the same name logic as the other files it makes it easier for the PC Port since they are sorted by name so having one and unique name is easier for tracking Music down

**Review by PurpleHato — CHANGES_REQUESTED** [review]

**Comment by NewSoupVi** [comment]
"Fanfare - Get quest item" hits the upper note limit for the piano instrument.
Was this fanfare previously accepted into the project? If so, I will accept it again. Otherwise, if it's a new submission, I'd rather that be fixed

**Comment by NewSoupVi** [comment]
Spirit Tracks Title Screen could be considered a bit loud in some parts, but I think it's not bad enough to block the merge.

**Pull request merged**